build-system/tasks/visual-diff.js

```diff
@@ -58,7 +58,9 @@ function extractPercyKeys() {
         'Error: PERCY_TOKEN doesn\'t look like a valid Percy API key'));
     process.exit(1);
   }
-  util.log('Percy token: ', util.colors.magenta('<redacted>'));
+  // Note: The secret percy token only gets printed during local use. Travis
```

Comment:
Only engineers who want to do a local visual diff run, who already have access to the tokens. Those who access the public travis logs will not see the token in the logs.